adapters/handlers/rest/authz/handlers_authz.go

```diff
@@ -427,6 +443,12 @@ func (h *authZHandlers) revokeRole(params authz.RevokeRoleParams, principal *mod
 		return authz.NewRevokeRoleBadRequest().WithPayload(cerrors.ErrPayloadFromSingleErr(fmt.Errorf("user id can not be empty")))
 	}
 
+	for _, role := range params.Body.Roles {
+		if strings.TrimSpace(role) == "" {
+			return authz.NewRevokeRoleBadRequest().WithPayload(cerrors.ErrPayloadFromSingleErr(fmt.Errorf("one or more of the roles you want to revoke is empty")))
+		}
+	}
```

Comment:
Is this needed? Wouldn't we just end up with "user not found" if I pass in " "? Same for the other endpoint

Reply:
That is for the passed roles to revoke or assign

Reply:
oki - but wouldn't it be the same there? We return that the role " " does not exist?

Reply:
" " is empty (invalid request body) , doesn't exists it requires authZ t